### PR TITLE
fix: proper content selection & use sitemap instead of crawling from root

### PR DESCRIPTION
**Pull request by harrysolovay**

### CODE REVIEW

File: configs/aws_amplify_new.json [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
+    "lvl5": "amplify-toc-contents h4, amplify-toc-content th",
+    "lvl6": "amplify-toc-contents h5",
+    "text": "amplify-toc-contents p, amplify-toc-contents li, amplify-toc-content td"
   },
   "js_render": true,
-  "js_wait": 2,
-  "conversation_id": [
-    "1115766863"
-  ],
+  "js_wait": 10,
```
Comment (s-pace) [inline configs/aws_amplify_new.json:19]: Way too big. Please do server side rendering instead

File: configs/aws_amplify_new.json [hunk 2]
```diff
@@ -1,28 +1,22 @@
 {
   "index_name": "aws_amplify_new",
-  "start_urls": [
-    "https://docs.amplify.aws/",
```
Comment (s-pace) [inline configs/aws_amplify_new.json:4]: Please keep it

File: configs/aws_amplify_new.json (hunk 1 above)
Comment (harrysolovay) [reply configs/aws_amplify_new.json]: Reverted it to 2

File: configs/aws_amplify_new.json (hunk 2 above)
Comment (harrysolovay) [reply configs/aws_amplify_new.json:4]: Put it back in place

**Comment by harrysolovay** [comment]
@s-pace please let me know if there're any other changes that need to be made. Thank you!

**Pull request merged**